packages/big-design/src/components/TableNext/Row/styled.tsx

```diff
@@ -20,4 +20,12 @@ export const StyledTableRow = styled.tr<StyledTableRowProps>`
   }
 `;
 
+export const StyledExpandedIcon = styled.button`
```

Comment:
Is there a reason why we are not using our private [`StylableButton`](https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Button/Button.tsx#L50)? That way we can use our `Button` props.

Reply:
Tried with the public one, but didn't know we could use that one ^  😓 , so decided to create a new one because I was having issues with the styles, but good to know that this one is more flexible.

Also the `StylableButton` adds styles when button is active, focused,  and hover. I guess I'm gonna leave these styles for now to have better accessibility even-though that is not specified in the designs, I think we could change it later in case this is not required.

<img width="1041" alt="Screen Shot 2022-07-21 at 1 33 19 PM" src="https://user-images.githubusercontent.com/39739180/180287964-710854ba-bbd9-4a91-a572-ee89a20942a0.png">